index.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-next-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {HelloWorldApp} from './src/application';
+
+(async function main() {
+  const app = new HelloWorldApp();
+  await app.start();
+})();
```

Comment:
You need to catch startup errors here, see the recently landed https://github.com/strongloop/loopback-next-hello-world/pull/25.